src/backend/api/__init__.py

```diff
@@ -68,6 +70,10 @@ def access_key(self) -> str:
     def secret_key(self) -> str:
         return self._secret_key
 
+    @property
+    def addressing_style(self) -> Literal["auto", "virtual", "path"]:
```

Comment:
Because Boto3 is specifying it the same way.
```
class _S3Dict(TypedDict, total=False):
    use_accelerate_endpoint: bool
    payload_signing_enabled: bool
    addressing_style: Literal["auto", "virtual", "path"]
    us_east_1_regional_endpoint: Literal["regional", "legacy"]
```

Reply:
Fair, but we could still be using an enum and doing a translation from enum to a Literal. Won't hold the PR on that though.

Reply:
Done